vir/Cargo.toml

```diff
@@ -10,9 +10,11 @@ serde = { version = "1.0", features = ["derive"] }
 rsmt2 = { git = "https://github.com/vakaras/rsmt2", branch = "labels" }
 thiserror = "1.0.23"
 derivative = "2.1.3"
+regex = "1.5"
 syn = { version = "1.0", features = ["full", "fold", "parsing", "derive"] }
 quote = "1.0"
 proc-macro2 = { version = "1.0", features = ["span-locations"] }
+prusti-common = { path = "../prusti-common" }
```

Comment:
We need to get rid of this dependency because `prusti-common` will depend on `vir`.